cmd/authctl/user/user.go

```diff
@@ -0,0 +1,42 @@
+// Package user provides utilities for managing user operations.
+package user
+
+import (
+	"fmt"
+	"os"
+
+	"github.com/spf13/cobra"
+	"github.com/ubuntu/authd/internal/consts"
+	"github.com/ubuntu/authd/internal/proto/authd"
+	"google.golang.org/grpc"
+	"google.golang.org/grpc/credentials/insecure"
+)
+
+// UserCmd is a command to perform user-related operations.
+var UserCmd = &cobra.Command{
+	Use:   "user",
+	Short: "Commands related to users",
+	Args:  cobra.NoArgs,
+	RunE:  func(cmd *cobra.Command, args []string) error { return cmd.Usage() },
+}
+
+// NewUserServiceClient creates and returns a new UserServiceClient.
+func NewUserServiceClient() (authd.UserServiceClient, error) {
+	authdSocket := os.Getenv("AUTHD_SOCKET")
+	if authdSocket == "" {
+		authdSocket = "unix://" + consts.DefaultSocketPath
+	}
```

Comment:
I would append the `unix://` thingy by default here, since it's what we do everywhere else (pam clients, daemon itself...) so I'd not make authctl special in this regards

Reply:
> We could prefix it with `unix://` if there is no scheme already

Ok, that's fine if you like it better.

Reply:
The prefixing thing isn't there yet, right?